tests/testthat/test-tmb-methods.R

```diff
@@ -393,7 +463,11 @@ test_that("model.frame do not care about subject levels", {
   out_frame <- expect_silent(
     model.frame(fit1, na.action = "na.fail", data = fev_data2, include = "subject_var")
   )
-  expect_identical(fev_data2$USUBJID, out_frame$USUBJID)
+  expect_identical(
```

Comment:
so this is the only change visible to the user. Can you please add `NEWS` file entries, both for this behavior change as well as additional methods?

Reply:
NEWS has been updated